tests/safemigrate_test.py

```diff
@@ -66,11 +67,17 @@ def test_backward(self, receiver):
         with pytest.raises(CommandError):
             receiver(plan=plan)
 
-    def default_after(self, receiver):
-        """Migrations are after by default."""
-        plan = [(Migration(), False)]
+    def test_default_always_wont_block(self, receiver):
+        """Migrations are safe always by default."""
+        plan = [(Migration(), False), (Migration(safe=Safe.after_deploy()), False)]
```

Comment:
From my understanding blocking occurs when you have `after_deploy` in front of `before_deploy`. If that's true, then the second migration here should be `before_deploy` to match the test's name, `test_default_always_wont_block`

Reply:
Good catch. I got the order of migrations wrong on one this one; it's supposed to block if the default is before_deploy.